src/js/gameController.js

```diff
@@ -0,0 +1,38 @@
+export default class Controller {
+    constructor(gameModel, gameView) {
+        this.gameModel = gameModel;
+        this.gameView = gameView;
+    }
+    init() {
+        this.gameView.boardCells.forEach(cell => {
+            cell.addEventListener("click", () => {
+                if (!this.gameModel.findWinningCombinations() && cell.innerText === '') {
+                    this.gameView.gameStatus.innerText = "In Progress"
+                    cell.innerText = this.gameModel.turn;
+                    this.gameModel.makeMove(cell.dataset.index);
+                    this.gameView.playerTurn.innerText = this.gameModel.turn + "'s Turn";
+                    if (this.gameModel.winnigPattern !== null) {
+                        this.gameModel.winnigPattern.forEach(pattern => {
+                            this.gameView.boardCells[pattern].classList.add("winner");
+                            this.gameView.gameStatus.innerText = this.gameView.boardCells[pattern].innerText + " Wins Click Restart Button For New Game";
+                        })
+                    }
+                    if (!this.gameModel.findWinningCombinations() && this.gameModel.board.every(item => item !== null)) {

```

Comment:
`this.gameModel.board.every(item => item !== null)` belə etmək olmaz, classlarda bunun üçün ayrıca funksiya yazmaq lazımdı setter və getter və ya clear()